dashboards/mintel/experimental/kubernetes-cluster-health-dashboard.json

```diff
@@ -382,31 +383,21 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "1 + (absent(count(up{job=~\"kubelet|etcd|kube-state-metrics|node-exporter\"} == 0)) or vector(0))",
+          "expr": "sum(kube_node_status_condition{condition!=\"Ready\", status=\"true\"} )",
           "format": "time_series",
           "instant": true,
+          "interval": "",
           "intervalFactor": 1,
           "legendFormat": "",
           "refId": "A",
           "step": 600
         }
       ],
-      "thresholds": "1,2",
-      "title": "Core Targets down ALERT",
+      "thresholds": "1,100",
+      "title": "Bad Node Conditions",
```

Comment:
Should this be `Nodes Not Ready` ? because we're testing the ready condition

Reply:
we are actually testing for `any condition` other than ready , they should all be false for a good node

Reply:
We're checking to see if any of the other conditions apart from "ready" are true.